.github/actions/install-zig/action.yml

```diff
@@ -0,0 +1,47 @@
+name: 'install-zig'
+description: 'Install zig compiler and make it available in PATH.'
+
+inputs:
+  sudo:
+    description: 'Specify a sudo command. Put it empty when sudo is not available.'
+    required: false
+    default: 'sudo'
+
+outputs:
+  zig_path:
+    description: "Path of installed zig, automatically added to GITHUB_PATH"
+    value: ${{ steps.zig.outputs.zig-path }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install zig
+      shell: bash
+      id: zig
+      env:
+        ZIG_VERSION: '0.14.0-dev.1588+2111f4c38'
```

Comment:
We need to use a development version of `zig` since it needs my patch: https://github.com/ziglang/zig/pull/21253
Hopefully they'll release 0.14.0 soon.

Also, keeping this PR wip as master zig artifacts are not kept around, ie: once a new master artifact is released, the old one gets removed.

Reply:
As a workaround, i am using `actions/cache` to cache the zig toolchain.